manifest.yml

```diff
@@ -1,8 +1,8 @@
 manifest_version: 1
-name: my dummy executor
-alias: MyDummyExecutor
-description: This is my dummy executor
+name: LaserEncoder
+alias: LaserEncoder
+description: encodes data from an 1d array of string in size `B` into an ndarray in size `B x D`.
 author: Jina AI Dev-Team (dev-team@jina.ai)
-url: https://github.com/jina-ai/EXECUTOR_REPO_NAME
+url: https://github.com/jina-ai/executor-text-laser-encoder
 avatar: https://avatars.githubusercontent.com/u/60539444?s=200&v=4
-keywords: [encoder, Numpy]
+keywords: [nlp, encoder, LASER, multilingual]
```

Comment:
according to the guideline, model name should be lowercased, and nlp should be text

Reply:
got it, thanks!